src/managers/organization/authentication-by-peers.ts

```diff
@@ -12,16 +12,72 @@ import {
 } from '../../config/errors';
 import { sendMail } from '../../connectors/sendinblue';
 import { updateUserOrganizationLink } from '../../repositories/organization/setters';
-import { getOrganizationsByUserId } from './main';
+import { getOrganizationById, getOrganizationsByUserId } from './main';
 import {
   createModeration,
   findPendingModeration,
 } from '../../repositories/moderation';
 import {
+  DEFAULT_MEMBER_COUNT_THRESHOLD_TO_ACTIVATE_SPONSORSHIP,
   NOTIFY_ALL_MEMBER_LIMIT,
   SUPPORT_EMAIL_ADDRESS,
 } from '../../config/env';
 
+export const isEligibleToSponsorship = async ({
+  id,
+}: Organization): Promise<boolean> => {
+  const organization = await getOrganizationById(id);
+  if (!organization) {
+    return false;
```

Comment:
If we are not able to find an organization, wouldn't be neater to return a 404? If we just return false in this case, we can think the organization exists but is not eligible.

Reply:
The managers do not handle requests, the controllers do.